charts/firefly/templates/_helpers.tpl

```diff
@@ -138,10 +138,12 @@ debug:
 {{- end }}
 http:
   port: {{ .Values.core.service.httpPort }}
+  publicURL: {{ .Values.core.service.httpPublicURL }}
```

Comment:
Not sure if this is the right place in the values hierarchy or not

Reply:
We should probably leave an option for override this value, but for now would like the chart to default to setting this based on the Service or Ingress hostnames.